config/initializers/money.rb

```diff
@@ -0,0 +1,116 @@
+# encoding : utf-8
+Money.locale_backend = :i18n
+
+MoneyRails.configure do |config|
+
+  # To set the default currency
+  #
+  config.default_currency = :eur
```

Comment:
Looks correct to me. you've included the most up-to-date version :i18n and have done this correctly.